src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -329,6 +329,11 @@ namespace nupic {
            */
           virtual UInt persistentSize() const;
 
+
+          UInt getPrecision() const {
```

Comment:
Ok, I think it should stay a method (something like class method in py), I want to be able to call `SP.getPrecision()` (and compare these for cpp & py impl.)